app/monitoring/create-failure.js

```diff
@@ -1,10 +1,13 @@
 const db = require('../data')
 
-const createFailure = async (deliveryId, reason, transaction) => {
+const createFailure = async (deliveryId, errorObject, timestamp, transaction) => {
   await db.failure.create({
-    deliveryId,
-    reason,
-    failed: new Date()
+    deliveryId: deliveryId,
```

Comment:
believe this one can be reverted back to just deliveryId for the shorthand. I think standardjs should be moaning about this